pkg/repo/mysql/translation.go

```diff
@@ -78,49 +78,49 @@ ON DUPLICATE KEY UPDATE
 	status = VALUES(status)`,
 	)
 	if err != nil {
-		return fmt.Errorf("repo: prepare stmt to insert message_message: %w", err)
+		return fmt.Errorf("repo: prepare stmt to insert message: %w", err)
 	}
 	defer stmt.Close()
 
-	for _, m := range messages.Messages {
+	for _, m := range translation.Messages {
 		_, err = stmt.ExecContext(
 			ctx,
-			messageID,
+			translationID,
 			m.ID,
 			m.Message,
 			m.Description,
 			m.Positions,
 			m.Status,
 		)
 		if err != nil {
-			return fmt.Errorf("repo: insert message_message: %w", err)
+			return fmt.Errorf("repo: insert message: %w", err)
 		}
 	}
 
 	if err := tx.Commit(); err != nil {
-		return fmt.Errorf("repo: commit tx to save messages: %w", err)
+		return fmt.Errorf("repo: commit tx to save translation: %w", err)
 	}
 
 	return nil
 }
 
-func (r *Repo) LoadMessages(ctx context.Context, serviceID uuid.UUID, opts repo.LoadMessagesOpts,
-) (model.MessagesSlice, error) {
+func (r *Repo) LoadTranslations(ctx context.Context, serviceID uuid.UUID, opts repo.LoadTranslationsOpts,
+) (model.Translations, error) {
 	rows, err := sq.
 		Select("mm.id, mm.message, mm.description, mm.positions, mm.status, m.language, m.original").
-		From("message_message mm").
-		Join("message m ON m.id = mm.message_id").
+		From("message mm").
+		Join("translation m ON m.id = mm.translation_id").
```

Comment:
All aliases have to change too.

```suggestion
		From("message m").
		Join("translation t ON t.id = m.translation_id").
```